tests/repro_subagent_anim_leak.py

```
@@ -0,0 +1,163 @@
+            return 1
+        else:
+            print("\n--- FIX VERIFIED: animation stopped after Ctrl+C ---")
+            return 0
+
+    finally:
+        try:
+            os.write(master, b"\x03\x03")
+            proc.terminate()
+            proc.wait(timeout=5)
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)